TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -397,10 +397,6 @@ public function createTagAction($label)
         $existingTag = $this->tagRepository->findByLabel($label);
```

Comment:
The issue is that `findBy..` returns a `QueryResult` (i.e. a *collection* of tags). It should be changed to
```
$existingTag = $this->tagRepository->findOneByLabel($label);
if ($existingTag !== null) {
 // ...
```